var/spack/repos/builtin/packages/rpm/package.py

```diff
@@ -60,8 +60,8 @@ class Rpm(AutotoolsPackage):
     # Always required
     depends_on("popt")
 
-    # Without this file patch, we don't detect lua
-    depends_on("lua+pcfile@5.3.5:", when="+lua")
+    # Without pc file, we don't detect lua
```

Comment:
I think the comment can now be removed, and the dependency on `lua` can be treated as any other dependency.